### PR TITLE
Merge release 1.12 into release 1.13

### PR DESCRIPTION
**Pull request by chenrui333**
Capture the contributors' commits during the release migration
relates to https://github.com/kubernetes/website/pull/14014

- PR from `release-1.12` to `release-1.13`
- Resolve conflicts
- Filter out the non-zh contents

cc @fleeto @markthink 

Original attempt PR: #14075

### CODE REVIEW

**Comment by chenrui333** [comment]
My original idea was to promote the changes from release-1.12 to release-1.13, and then from release-1.13 to release-1.14. And create tasks on top of release-1.14. 

Since the major work was done in another PR, maybe we just to double-check and recreate a small patch PR to release-1.14.

From content-zh perspective, release-1.13 and release-1.14 should be pretty much the same (no work has ever been done either of those releases).

**Comment by chenrui333** [comment]
```
9:32:12 PM: Total in 1783 ms
9:32:12 PM: Error: Error building site: "/opt/build/repo/content/zh/docs/reference/setup-tools/kubeadm/generated/README.md:2:1": invalid YAML delimiter
9:32:12 PM: Error running command: Build script returned non-zero exit code: 255
9:32:12 PM: Failing build: Failed to build site
9:32:12 PM: failed during stage 'building site': Build script returned non-zero exit code: 255
9:32:12 PM: Finished processing build request in 40.246459256s
9:32:12 PM: Shutting down logging, 0 messages pending
```

**Comment by chenrui333** [comment]
```
10:53:12 PM: Error: Error building site: "/opt/build/repo/content/zh/docs/reference/setup-tools/kubeadm/generated/README.md:2:1": starting HTML comment with no end
10:53:12 PM: Error running command: Build script returned non-zero exit code: 255
10:53:12 PM: Failing build: Failed to build site
10:53:12 PM: failed during stage 'building site': Build script returned non-zero exit code: 255
10:53:12 PM: Finished processing build request in 1m9.780047701s
10:53:12 PM: Shutting down logging, 0 messages pending
```

**Comment by chenrui333** [comment]
Will just chose to remove that md file for simplicity. 😄

**Comment by tengqm** [comment]
/hold 
@chenrui333 is this ready?

**Comment by SataQiu** [comment]
One tip @chenrui333 :
You need to remove the files that are not in `content/zh`.
And then we can merge it.
图片

**Comment by chenrui333** [comment]
@tengqm yeah, it is almost ready besides removing some new files.

@SataQiu I will remove those new files.

**Comment by chenrui333** [comment]
@SataQiu Removed the generated sass assets.

Also, a quick question, do I need to remove all the contents show as added?

**Comment by SataQiu** [comment]
@chenrui333 
We just need to make sure we don't change anything other than the `content/zh`.
I think that's enough.

File: content/zh/blog/_posts/2015-04-00-Kubernetes-Release-0150.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+* Added rate limiting to pod deleting ([#6355][12])
+* Implement Balanced Resource Allocation algorithm as a PriorityFunction in scheduler package ([#6150][13])
+* Enabled log collection from master ([#6396][14])
+* Added an api endpoint to pull logs from Pods ([#6497][15])
+* Added latency metrics to scheduler ([#6368][16])
+* Added latency metrics to REST client ([#6409][17])
+
+-->
+
+* 启用 1beta3 API 并将其设置为默认 API 版本 ([#6098][1])
```
Comment (zhangxiaoyu-zidif) [inline content/zh/blog/_posts/2015-04-00-Kubernetes-Release-0150.md]: typo： v1beta3

**Comment by zhangxiaoyu-zidif** [comment]
It's really not easy to review... :(
some nits could be patched later.

File: content/zh/blog/_posts/2015-04-00-Kubernetes-Release-0150.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/blog/_posts/2015-04-00-Kubernetes-Release-0150.md]: Do we need to address this per the PR?

**Comment by chenrui333** [comment]
@SataQiu @xichengliudui @tengqm @markthink @zhangxiaoyu-zidif
ping for merg! 😄

**Comment by chenrui333** [comment]
@SataQiu @xichengliudui @tengqm @markthink @zhangxiaoyu-zidif
ping for merge! 😄

**Comment by chenrui333** [comment]
@xichengliudui that is probably not the right prow command 😆

**Comment by xichengliudui** [comment]
can try to resubmit and add the submission information.

**Pull request merged**